src/dsp_tools/utils/id_to_iri.py

```diff
@@ -84,7 +84,7 @@ def id_to_iri(
         out_file = file_name + "_replaced_" + timestamp_str + ".xml"
 
     et = etree.ElementTree(tree.getroot())
-    et.write(out_file, pretty_print=True)
+    et.write(out_file, pretty_print=True, xml_declaration=True, encoding="utf-8")
```

Comment:
all other DSP XML files have a declaration, so it's probably better to add one here, too.

Reply:
I'm surprised this isn't done by default. It's always better to add the declaration